python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/ptp.schema.yml

```
@@ -60,6 +60,15 @@ keys:
         max: 255
         convert_types:
         - str
+      hold_ptp_time:
+        type: int
+        min: 0
+        max: 172000
+        description: |-
+          The length of time that PTP will remain in hold-down on the internal oscillator if the connection to the GM is lost.
+          If a new GM is not elected before the expiry of the hold-down time, the clock will revert to free-running.
+        convert_types:
+          - str
```

```suggestion
        convert_types:
        - str
```
Apologies for confusion.